src/commands/bf.mexists.json

```
@@ -0,0 +1,26 @@
+{
+    "BF.MEXISTS": {
+        "summary": "Determines if the bloom filter contains one or more items",
+        "complexity": "O(k * n), where n is the number of hash functions used by the bloom filter and k is the number of items",
```

nit: Could we put the symbol(s) in time complexity in upper caps?

```suggestion
        "complexity": "O(N * K), where N is the number of hash functions used by the bloom filter and K is the number of items",
```

---

Will update for all commands